File: python/animals/wilk.py
from animals.zwierze import Zwierze


class Wilk(Zwierze):

    def __str__(self):
        return self._species + "[" + str(self._x) + "][" + str(self._y) + "]"

    def __init__(self, x, y, world, sila=9):
        Zwierze.__init__(self, "wilk", sila, 4, x, y, world)

    def rysowanie(self):
        return "W"

File: java/src/Zwierzeta/Wilk.java
package Zwierzeta;
import Swiat.*;

public class Wilk extends Zwierze{
    public Wilk(int x, int y , Swiat swiat){
        super(Gatunek.WILK, 9,5, x, y, swiat);
    }
    public Wilk(int x, int y, int sila, Swiat swiat){
        super(Gatunek.WILK, sila,5, x, y, swiat);
    }

    @Override
    public String rysowanie(){return "W";}
}
